src/var.c

```diff
@@ -250,11 +269,58 @@ static int fi_parse_bool(const char *str_value)
 	return -1;
 }
 
+static void load_conf() {
+	char *line = NULL;
+	char *tmp;
+	struct ofi_conf_entry *conf;
+	size_t length = 0;
+	ssize_t read;
```

Comment:
I'd pick a different name than one that collides with a standard function name.  Maybe bytes?